### ISSUE TITLE
[Question]: #find command expected behavior

### ISSUE THREAD

**Issue by Deadman0001** [Charismara/pixelmon-hacks-issues#41]
### Question

With the new addition of the `Form` argument to the #find command, regional forms of pokemon are now a different search from the normal form. If you specify the search further to the regional variant, the #find command works as expected.

For example, in previous versions of Pixelmon Hacks, if I do the command, `#find add Rattata`, it will place a waypoint to all the Rattata's around me. However, in the newest version of 2.1.16.3, it will only place waypoint to a normal Rattata and all palettes except the different regional forms. To find an Alolan Rattata, you would have to instead have to do the command, `#find add Rattata Alolan`

This applies to Alolan, Galarian, Hisuian, and Paldean forms. I was wondering if this is now expected behavior.
Attached are images of my testing.

![find_regional_forms](https://github.com/Charismara/pixelmon-hacks-issues/assets/134663392/fa2a8572-e7f5-4d45-a90a-f9afb62ec80d)
![find_regional_forms2](https://github.com/Charismara/pixelmon-hacks-issues/assets/134663392/faa7a68c-e0ad-4c1f-9184-e1cc86345ec8)
![find_regional_forms3](https://github.com/Charismara/pixelmon-hacks-issues/assets/134663392/e33cb53d-e786-4592-a32a-ba46fa6cd57d)
![find_regional_forms4](https://github.com/Charismara/pixelmon-hacks-issues/assets/134663392/e250ef22-ce98-4549-95ad-0f6591d21c80)

**Comment by Deadman0001** [comment]
On further investigation, this doesn't just apply to regional forms, it just applies to any pokemon with additional forms in general.

For example, Aegislash Shield and Aegislash Blade are two different pokemon according to #find
Additional examples: 
- All the different Lunatone forms
- Lycanroc forms
- Hoopa forms
- Meloette forms
- Mimikyu forms
- Minior meteor vs core forms
- Necrozma forms
- Oricorio forms
- Sinistea/Polteageist forms
- Rotom forms
- Shaymin forms
- Silvally forms
- etc

**Comment by Deadman0001** [comment]
An additional bug I found while testing is on some pokemon, you cannot remove them from the #find list. I'm not sure of all the pokemon that cannot be removed but so far I found that Aegislash None, Lycanroc None, Lunatone None can't be removed. The entire list can be purged with `#find clear` however.

![find_remove_bug](https://github.com/Charismara/pixelmon-hacks-issues/assets/134663392/fc407ece-d2c2-40b9-ae06-07d3a865961d)
